src/Session.php

```diff
@@ -105,7 +105,7 @@ public function hasSection($section)
 
 	public function getIterator()
 	{
-		return new \ArrayIterator();
+		return new \ArrayIterator(array_keys($this->sections));
```

Comment:
This is intended to be compatible with http://api.nette.org/2.2.7/source-Http.Session.php.html#302-318. I didn't test it though.